client/src/app/users/add-user.component.ts

```diff
@@ -12,7 +12,7 @@ import { UserService } from './user.service';
 })
 export class AddUserComponent implements OnInit {
 
-  addUserForm: FormGroup;
+  addUserForm: UntypedFormGroup;
```

Comment:
Thinking about it, we almost certainly want to move to typed form groups since then the type checker can help us out more. We should probably create an issue for that.

Reply:
I agree; It just happened to not seem broken when it was just listed as being a FormGroup... but I'm not sure if that's meaningful TBH. So, I went ahead and made this a noticeable thing to address.